fetch.bs

```diff
@@ -2049,6 +2051,29 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partition>HTTP Cache partition</h3>
+
+<p>To determine the associated <dfn export id=http-cache>HTTP cache</dfn> partition, given
+<var>request</var>, run these steps:
+
+<ol>
+ <li>
+  <p>If <var>request</var>'s <a for=request>client</a> is null, terminate these steps.</li>
```

Comment:
You can't terminate the steps without returning something. Maybe return null? Per https://github.com/whatwg/html/pull/4966#issuecomment-577316426 it sounds like in that case nothing will be cached, so, you'd probably need to update the processing model to account for when the return value of this algorithm is null.

Reply:
Done.